frame/asset-conversion/src/lib.rs

```diff
@@ -350,6 +350,8 @@ pub mod pallet {
 		PathError,
 		/// The provided path must consists of unique assets.
 		NonUniquePath,
+		/// It was not possible to get and increment the pool asset id.
+		IncorrectPoolAssetId,
```

Comment:
Maybe something more descriptive here would be nice - like `UnableToIncrementAssetId` or `AssetIdIncrementationError` ?

Reply:
Would be nice to have a test for when this error is expected.